xonsh/tools.py

```diff
@@ -633,17 +617,17 @@ def bool_seq_to_csv(x):
 
 
 def is_completions_display_value(x):
-    return x in {'none', 'single', 'multi'}
+    return x in ('none', 'single', 'multi')
```

Comment:
uniqueness is guaranteed. if one wrote `('none', 'single', 'none')` here that'd be an obvious bug. an immutable saves some resources.